scan-command/src/test/java/io/ballerina/scan/utils/ScanUtilsTest.java

```diff
@@ -126,4 +118,112 @@ void testGenerateScanReportToProvidedDirectory() throws IOException {
                 .replace(WINDOWS_LINE_SEPARATOR, LINUX_LINE_SEPARATOR);
         Assert.assertEquals(result, expected);
     }
+
+    @Test(description =
+            "test method for loading configurations from a scan toml file in a Ballerina single file project")
+    void testloadScanTomlConfigurationsForSingleFileProject() {
```

Comment:
Are we going to add tests where the Scan.toml is resolved using an URL and also negative test cases, where the Scan.toml is not present and so on here?

Reply:
For testing the `Scan.toml` file resolved from a URL, a remote configuration file needs to exist. So once this PR is merged will be adding tests for loading remote `Scan.toml` configurations by pointing to the file within this repository.